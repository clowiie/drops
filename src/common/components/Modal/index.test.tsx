import React from 'react'

import { render } from '@testing-library/react'

import * as ModalType from '.'

describe('Modal', () => {
  const onCloseSpy = jest.fn()
  const children = <div>Content</div>

  const { default: Modal } = require('.') as typeof ModalType

  describe('snapshots', () => {
    it('should match snapshots', () => {
      const { container } = render(
        <Modal isOpen onClose={onCloseSpy}>
          {children}
        </Modal>,
      )

      expect(container.firstChild).toMatchSnapshot()
    })
  })
})
