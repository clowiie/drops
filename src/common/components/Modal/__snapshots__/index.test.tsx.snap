// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal snapshots should match snapshots 1`] = `null`;
